docs/README.md

```diff
@@ -0,0 +1,214 @@
+# Integrating Kobiton into CircleCI mobile application development pipeline
+## Table of contents
+- [A. Integrating Kobiton with CircleCI](#a-integrating-kobiton-with-circleci)
+    - [1. Preparation](#1-preparation)
+    - [2. Setup](#2-setup)
+        - [2.1. Setting Kobiton Username and API Key](#21-setting-kobiton-username-and-api-key)
+        - [2.2. Setting Kobiton device desired capabilities](#22-setting-kobiton-device-desired-capabilities)
+        - [2.3. CircleCI configuration file](#23-circleci-configuration-file)
+    - [3. Automation Test Execution](#3-automation-test-execution)
+- [B. Test session details](#b-test-session-details)
+    - [1. Viewing test session details on Kobiton website](#1-viewing-test-session-details-on-kobiton-website)
+    - [2. Fetching test session details using Kobiton REST API](#2-fetching-test-session-details-using-kobiton-rest-api)
+- [C. Feedback](#c-feedback)
+
+## A. Integrating Kobiton with CircleCI
+### 1. Preparation
+#### 1.1. Getting Kobiton Username and API key
+Kobiton Username and API key are required for authenticating with Kobiton API.
+
+> If you don't have a Kobiton account, visit https://portal.kobiton.com/register to create one.
+
+To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script for Kobiton` section on [our blog](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/).
+
+#### 1.2. Samples
+To give you an in-depth demonstration of the integration flow, we have provided samples:
+- Script for executing automation test on Kobiton devices : [automation-test-script.js](/samples/automation-test/automation-test-script.js).
+- CircleCI configuration file : [config.yml](/.circleci/config.yml).
+
+Follow steps below to get started:
```

Comment:
```
Follow steps below to get started:
 1. Fork this repository.
2. Synchronize the forked repository with CircleCI.
> Refer to [CircleCI documentation](https://circleci.com/docs/2.0/getting-started/) for instructions on how to synchronize a repository with CircleCI.
>
> Note : At first synchronization, the build process will be failed because we haven't done any configuration. This is normal, just ignore the `Build Failed` message.
```

All above should be at 2.1, and the old 2.1 is pushed to 2.2.